docs/build.md

```diff
@@ -0,0 +1,17 @@
+# Build and test
+
+Please make sure Rust is [installed](https://www.rust-lang.org/learn/get-started).
+
+**Basic**
+
+Run `cargo build` or `cargo test` to get started with the Fe compiler.
+
+**Full**
+
+The Fe compiler depends on the Solidty compiler for transforming Yul IR to EVM bytecode. We currently use [solc-rust](https://github.com/axic/solc-rust) to perform this. In order to compile solc-rust, the following must be installed on your system:
+
+- cmake
+- libboost
+- libclang
+
+Once these have been installed, you can run `cargo build --features solc-backend` to build the full Fe compiler. This may take up to an hour.
```

Comment:
I think this needs to be `cargo build --workspace --features solc-backend` when given from the root.

Reply:
I think this part is ok, since we're building the CLI, which depends on the other libraries.

We should probably add another line though that tells them how to test the whole project, which would be:

`cargo test --workspace --features solc-backend`

Reply:
Ah, absolutely. I was *thinking*  about running the tests when in fact this is only about building the project :man_facepalming: